python/ray/_private/usage/usage_lib.py

```diff
@@ -182,6 +193,106 @@ def put_cluster_metadata(gcs_client, num_retries) -> None:
     return metadata
 
 
+def get_cluster_status(gcs_client, num_retries) -> dict:
```

Comment:
cc @wuisawesome for awareness of additional uses of the DEBUG_AUTOSCALING_STATUS message
(the code here looks fine -- we're planning a refactor of how autoscaling status info is i/o'd between ray internals and the rest of the world)